SConstruct

```diff
@@ -312,10 +296,7 @@ if env["import_env_vars"]:
 
 # Platform selection: validate input, and add options.
 
-if env.scons_version < (4, 3) and not env["platform"]:
-    env["platform"] = env["p"]
-
-if env["platform"] == "":
+if not env["platform"]:
```

Comment:
That's no big deal but with the ambiguity between `None` and `""` both evaluating as false, I tend to prefer being explicit in Python. I don't feel too strongly about it though.

Reply:
I'm personally a pretty big fan of the truthy/falsy styled checks, at least in dynamic environments like Python. Not to mention that there's no scenario where `env["platform"]` would be `None` here

Reply:
>  Not to mention that there's no scenario where `env["platform"]` would be `None` here

I have to read the whole SConstruct to know that, and there's no guarantee that it wouldn't change in the future. So that's precisely why I think being explicit about what the type should be is important.